### ISSUE TITLE
Issue: kReachabilityChangedNotification ambiguous

### ISSUE THREAD

**Issue by benv33** [zoho/SalesIQ-Mobilisten-iOS#19]
**Describe the bug**
When trying to compile my app I get :
 _Reference to 'kReachabilityChangedNotification' is ambiguous_

**To Reproduce**
1. Create an app
2. Add   pod 'Mobilisten' and     pod 'Reachability' to pod file
3. Add headers for both project in a file and reference kReachabilityChangedNotification
4. Build fails with _Reference to 'kReachabilityChangedNotification' is ambiguous_

**Expected behavior**
The app should compile without error

**Screenshots**

**Smartphone (please complete the following information):**
- Do you usually run the project on the simulator or a physical device?
 - Device: Simulator iPhone 12 Pro Max
 - OS: iOS 14.5
 - Xcode Version: 12.5.1
 - Mobilisten Version: 4.1.0

**Additional context**
The constant kReachabilityChangedNotification is defined in Reachability and Mobilisten library

**Comment by Rishabh-Raghunath** [comment]
Hi @benv33, thanks for bringing this conflict between the Mobilisten and the Reachability pods to our notice.

```objc
NSString *const kReachabilityChangedNotification = @"kReachabilityChangedNotification";
```

Since **kReachabilityChangedNotification** is just a String, an easy workaround would be to add an observer using the same raw value.
Here's a quick example in Swift that would work.

```swift
extension Notification.Name {
    static let reachabilityChanged = Notification.Name("kReachabilityChangedNotification")
}

// Adding an observer for the reachability changed notification.
NotificationCenter.default.addObserver(self, selector: #selector(self.reachabilityChanged(notification:)), name: .reachabilityChanged, object: nil)
```

We'll definitely have this addressed through an update to the Mobilisten pod. Until then, you may use the above snippet to add an observer.

**Comment by Rishabh-Raghunath** [comment]
Hi @benv33, this issue has been addressed in Mobilisten **4.2.0**.
Please update Mobilisten to the latest version using Cocoapods by running the `pod update Mobilisten` command from the project directory.

Once again, thanks for bringing this issue to our notice.
